workflows_core/workflow/context_manager.py

```diff
@@ -89,28 +89,14 @@ def __enter__(self):
         return
 
     def __exit__(self, exc_type: type, exc_value: BaseException, traceback: Traceback):
+        self._calculate_pricing()
```

Comment:
will this calculate on failed workflows

Reply:
it might only work for 30% of documents - so we still want to calculate pricing if that's the case - although I need to adjust it to be based on the success threshold

Reply:
@jtwinrelevanceai  
I'll move it to only calculate on success for now - can you please ticket up to make it dependent on the success threshold actually?